routingtable/endpoint.go

```diff
@@ -95,22 +95,36 @@ func (info ExternalEndpointInfo) Hash() interface{} {
 	return info
 }
 
+func ptrOfInt(i int) *int { return &i }
```

Comment:
This change is not necessary to merge, but I would consider renaming this to `pointerToInt` or `ptrToInt` to make this more clear.

Reply:
Fixed